tests/test_scaffold.py

```diff
@@ -0,0 +1,80 @@
+import pathlib
+
+import pytest
+
+from pineko import configs, scaffold
+
+
+def construct_wrong_fake_configs(tmp_path):
```

Comment:
you could make those even fixtures (then, I think, you don't run in the risk of mutability)

Reply:
You can make a session-wise fixture

Reply:
https://docs.pytest.org/en/6.2.x/fixture.html#scope-sharing-fixtures-across-classes-modules-packages-or-session

Reply:
Thanks, I don't think this to be exactly what I need but I managed to find a solution :)